common/export-common-cartridge/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>common</artifactId>
+    <groupId>org.sakaiproject.common</groupId>
+    <version>11-SNAPSHOT</version><!-- ProjectVersion -->
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <name>Sakai Export Common Cartridge</name>
+  <groupId>org.sakaiproject.common</groupId>
+  <artifactId>export-common-cartridge</artifactId>
+  <organization>
+    <name>Sakai Foundation</name>
+    <url>http://sakaiproject.org/</url>
+  </organization>
+  <inceptionYear>2005</inceptionYear>
+	<packaging>jar</packaging>
+  <properties>
+	  <deploy.target>shared</deploy.target>
+  </properties>
+  <dependencies>
+	  <dependency>
```

Comment:
The indentation here looks wrong, could you try to match it with the surrounding context, the issue maybe because of differences tabs and spaces.